base_multi_company/models/multi_company_abstract.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class MultiCompanyAbstract(models.AbstractModel):
+
+    _name = 'multi.company.abstract'
+    _description = 'Multi-Company Abstract'
+
+    company_id = fields.Many2one(
+        string='Company',
+        comodel_name='res.company',
+        compute='_compute_company_id',
+        store=True,
```

Comment:
Why not? It's stored in the other modules and seems to be working fine:

* https://github.com/OCA/multi-company/blob/9.0/product_multi_company/models/product_template.py#L22
* https://github.com/OCA/multi-company/blob/9.0/partner_multi_company/models/res_partner.py#L20